x-pack/plugins/cases/public/components/user_actions/comment/alert_event.tsx

```diff
@@ -38,12 +38,16 @@ const RuleLink: React.FC<SingleAlertProps> = memo(
 
     const ruleDetailsHref = getRuleDetailsHref?.(ruleId);
     const finalRuleName = ruleName ?? i18n.UNKNOWN_RULE;
+    const isValidLink = useMemo(() => {
+      if (!onRuleDetailsClick && !ruleDetailsHref) return false;
```

Comment:
super nit: Could you please use `if` block  same as line 46, to make it more readable? 
```typescript
if (!onRuleDetailsClick && !ruleDetailsHref) { 
  return false;
} 
```